CMakeLists.txt

```diff
@@ -44,6 +44,8 @@ endif()
 # ---------------------------------------------------------------------------
 
 if(NB_CREATE_INSTALL_RULES AND NOT CMAKE_SKIP_INSTALL_RULES)
+  # Silence warning in GNUInstallDirs due to no enabled languages
+  set(CMAKE_INSTALL_LIBDIR "")
```

Comment:
Can you explain this line a bit more? The effect of this is not obvious to me.

Reply:
The comment above is attached to this (so that should be changed to provide more clarity), but if you remove this line, you will see a warning from the `GNUInstallDirs` CMake module.
```sh
$ cmake -B build -S . -DNB_TEST=OFF
-- nanobind: setting build type to 'Release' as none was specified.
CMake Warning (dev) at /opt/homebrew/Cellar/cmake/3.27.7/share/cmake/Modules/GNUInstallDirs.cmake:243 (message):
  Unable to determine default CMAKE_INSTALL_LIBDIR directory because no
  target architecture is known.  Please enable at least one language before
  including GNUInstallDirs.
Call Stack (most recent call first):
  CMakeLists.txt:48 (include)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Configuring done (0.0s)
-- Generating done (0.0s)
-- Build files have been written to: /Users/ekilmer/src/nanobind/build
```
Setting `CMAKE_INSTALL_LIBDIR` is, unfortunately, necessary here to silence the warning.

Reply:
The warning comes now because I've set `LANGUAGES NONE` in the `project` line.

Reply:
Aha! Got it, thanks!